django/db/models/manager.py

```diff
@@ -242,12 +242,15 @@ class Manager(BaseManager.from_queryset(QuerySet)):
 class ManagerDescriptor(object):
     # This class ensures managers aren't accessible via model instances.
     # For example, Poll.objects works, but poll_obj.objects raises AttributeError.
-    def __init__(self, manager):
+    def __init__(self, manager, model):
         self.manager = manager
+        self.model = model
 
     def __get__(self, instance, cls=None):
         if instance is not None:
             raise AttributeError("Manager isn't accessible via %s instances" % cls.__name__)
+        elif not cls == self.model and cls._meta.proxy is False:
```

Comment:
This will pass if you have a proxy for a concrete child model and try to access the parent model's custom manager. You need to check if `cls` is a direct proxy for `self.model`.